### DISCUSSION TITLE
캐릭터의 조이스틱을 변경하고 싶어요.

### DISCUSSION THREAD

**Discussion by hina9** [naverz/zepeto-studio-kor#736] [5. Q&A]
여기 있는 샘플 예제들을 보면 캐릭터 생성이 되고, 조이스틱이 자동으로 나와있으면서 그자리에 고정이 되어있는데,

https://studio.zepeto.me/guides/character-control 
공식 예제를 보고 직접 따라하였더니, 

화면의 절반 조금 넘는 화면이 전체 터치 스크린이고, 터치한 위치에 조이스틱이 표현되더라구요..

터치 스크린을 없애고, 조이스틱을 한 곳에 고정할 수 있는 방법이 있을까요?

**Comment by hina9** [comment]
https://user-images.githubusercontent.com/93582930/167977219-6d3399a7-119b-4c6b-bb34-e70ea48cbe39.mp4




![image](https://user-images.githubusercontent.com/93582930/167977568-10bef6c2-e0d3-4a75-87c2-7d16b213424e.png)

hierachy 내의 ZEPETO -> ZepetoPlayers 기본 사용 후 수정은 하지 않았습니다.

**Comment by minjujuu** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/673#discussioncomment-2651826

위 링크의 답변을 참고하시면 터치 스크린의 영역을 조절하실 수 있을 것 같습니다!
